README.md

```diff
@@ -33,12 +33,13 @@
 [discord-badge]: https://img.shields.io/discord/885562378132000778.svg?logo=discord&style=flat-square
 [discord-url]: https://discord.com/invite/Qw5gKqHxUM
 
-[Website](https://github.com/apache/datafusion) |
-[Guides](https://github.com/apache/datafusion/tree/main/docs) |
+[Website](https://datafusion.apache.org/) |
 [API Docs](https://docs.rs/datafusion/latest/datafusion/) |
 [Chat](https://discord.com/channels/885562378132000778/885562378132000781)
 
-<img src="./docs/source/_static/images/2x_bgwhite_original.png" width="512" alt="logo"/>
+<a href="https://datafusion.apache.org/">
```

Comment:
this makes it so when you click on the logo on the github repo you go to https://datafusion.apache.org/

If you do this on the current version of https://github.com/apache/datafusion?tab=readme-ov-file you go to 

https://github.com/apache/datafusion/blob/main/docs/source/_static/images/2x_bgwhite_original.png